linux_docker_resources/Dockerfile

```diff
@@ -125,7 +125,8 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   python3-pytest-timeout \
   python3-yaml \
   uncrustify \
-  yamllint
+  yamllint \
+  wget
```

Comment:
I think we don't need this change anymore, since we aren't using `wget` anymore.